foundations/javascript_basics/project_rock_paper_scissors.md

```diff
@@ -1,44 +1,80 @@
 
-### Quick exercises before starting
+<div class="lesson-note">
+Some of the student solutions below contain buttons, text, and other elements. These elements are part of what is called a graphical user interface (GUI). You'll create the GUI in a later lesson. In the meantime, remember to commit your code to GitHub.
+</div>
+
+### Problem solving approach
 
-1. Identify three ways to include JavaScript in a page.
-1. Test it out! Write `console.log("Hello World");` in JavaScript and check to see if it displays in the browser's console.
+Remember the previous wise words from the [Problem Solving lesson](https://www.theodinproject.com/lessons/foundations-problem-solving) because this is your first JavaScript program built from scratch. Before you start this project, follow these steps for each step of the project:
 
-Finally, this is your first JavaScript program built from scratch, so don't forget the previous lesson on problem solving. Plan your solution out before writing any code, and test each piece as you build to ensure it is working before moving on to the next!
+1. Plan or pseudocode your solution.
+1. Write the code.
+1. Test your code to make sure it works.
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
-Don't forget to commit early & often! You can [reference the Commit Message lesson here](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages)!
+Remember to commit early and often! To refresh your memory, check out the [commit messages lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/commit-messages).
+
+#### Step 1: Initial structure
+
+1. Create a new Git repository for your project.
+1. Create a blank HTML document with a script tag.
+1. Check if the webpage includes JavaScript:
+   - Write `console.log("Hello World")` in JavaScript.
+   - Check if "Hello World" is logged in the browser console once you open your webpage.
+
+It's best practice to link to an external JavaScript file inside this script tag. Using an external JavaScript file keeps your HTML file clean and organized.
+
+You don't have to write additional code in the HTML file. This game is played entirely via the console.
+
+#### Step 2: Get computer choice function
+
+Your game will be played against the computer. You will make a function that randomly returns 'Rock', 'Paper' or 'Scissors':
+
+1. Create a new function named `getComputerChoice`.
+1. Write the code so that `getComputerChoice` randomly `return` one of the following string values: "Rock", "Paper" or "Scissors".
+   - **Hint**: The [Math.random](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random) method returns a random number that's greater than or equal to 0 and less than 1. Think about how you can use this to conditionally return one of the multiple choices.
```

Comment:
Kind of feel like maybe this would be worth omitting, and just telling users to randomly return a string (and just say "This would be a great time to Google how to do so!" or something). That said, if this has been a pain point for users on Discord for this project I'm fine leaving this here.

Reply:
It has actually been somewhat of a pain point, surprisingly enough.
It's been somewhat often where just mentioning `Math.random` gets people on the right tracks but despite a brief mention of it earlier in the curriculum, a lot of people seem to be stuck at what to even think about looking into here.